lib/file/widgets/file_thumbnail.dart

```diff
@@ -43,7 +43,7 @@ class FileThumbnail extends StatelessWidget {
     if (file.isDirectory) {
       return Icon(Icons.folder);
     }
-    final type = file.name.substring(file.name.lastIndexOf('.') + 1);
+    final type = file.extension;
```

Comment:
Do we want to improve file type detection? https://pub.dev/packages/mime

Reply:
We already use the `mime` package for detecting the file type when uploading files, but in this case, it'll also require some more work of renaming the current file thumbnail assets. For example, the `.avi` extension has the mime type `video/x-msvideo`. So we should probably open another issue for this.

Reply:
→ #226